docs/blog_post.md

```diff
@@ -0,0 +1,56 @@
+DRAFT ONLY--DO NOT QUOTE
+
+# ShopRunner Data Science for Good: Automating Urban Wildlife Classification
+
+![coyote](https://github.com/uptake/autofocus/raw/master/gallery/coyote1.jpg) 
+<!--TK image caption formatting-->
+Software developed by data scientists at ShopRunner and Uptake supports the work of urban wildlife researchers by automatically classifying images taken by motion-activated cameras, such as this image of a coyote in the Chicago area.
+
+The growth of cities threatens animal species that do not adapt well to urban environments. **The Lincoln Park Zoo's Urban Wildlife Institute studies how urbanization affects wildlife in part by collecting images from motion-activated cameras**, beginning in 2010 around Chicago but now from more than twenty urban areas. For almost a decade, volunteers have been helping in their efforts by identifying the animal species that appear in each of those images in the online [Zooniverse](https://www.zooniverse.org/projects/zooniverse/chicago-wildlife-watch) crowdsourcing platform. However, the rate of image collection now exceeds the rate of image labeling. The information that the images contain is effectively unavailable to researchers until they have been labeled, so **image labeling has become a bottleneck that holds back their research.**
```

Comment:
I'd add just a little more info into this first paragraph. Suggestion below.

```suggestion
We live on an urban planet, and as cities continue to grow in size, a basic challenge for conservation is to understand how wildlife respond to urbanization.  **The Lincoln Park Zoo's Urban Wildlife Institute studies how urbanization affects wildlife by placing motion-activated cameras in urban green spaces**. When a species walks in view of the camera, a photo is taken, which can then be used to determine where in a city a species resides. And while their study started in Chicago in 2010, the Urban Wildlife Institute now collaborates with partners in over 20 cities to systematically collect data on urban wildlife distributions using the same motion-activated cameras. Scaling up this research has resulted in a huge increase in the number of photos to classify. Historically, volunteers have been helping in their efforts by identifying the animal species that appear in each of those images in the online [Zooniverse](https://www.zooniverse.org/projects/zooniverse/chicago-wildlife-watch) crowdsourcing platform. However, the rate of image collection now exceeds the rate of image labeling. The information that the images contain is effectively unavailable to researchers until they have been labeled, so **image labeling has become a bottleneck that holds back their research.**
```

Reply:
Thanks! Given that my main goal is to recruit more data scientists, I'm inclined to keep the current structure as it is for the sake of getting to the data science as quickly as possible. However, I will adopt some of the wording you've suggested and perhaps sprinkle in more context after the opening paragraph.